### ISSUE TITLE
UX Improvements

### ISSUE THREAD

**Issue by cedricium** [tryunearth/app#10]
u/FloatyFish came in clutch with suggestions on how to improve Unearth's UX. Below is the list they gave me, might break these into individual tickets in the future.

- [ ] Tagging - **SEE #5**
  - [ ] Autocomplete tags when adding them if possible.
  - [ ] When the tag dialog is open, have a few of the most recent tags in a section below the main tagging area, users can click on them to add them.
- [ ] Have sorting on homepage/comments (start with chronological).
- [ ] Scrollbar (vertical) doesn't seem to actually scroll if you drag it in the home section, but if you scroll using the scroll wheel on the mouse it'll scroll.
- [ ] Could use a Post category under filters.
- [ ] Searchbar on the homescreen - **SEE #7**
- [ ] Showing the last time you synced would be helpful since there is currently no autosync.
- [ ] Personally, I would have the trashcans for tags greyed out instead of hidden. This will help with discoverability. However, redditors are generally more tech savvy and may overlook that.
- [ ] When hovering over subreddits, have some sort of visual indicator. Text change is too subtle.
  - [ ] Reuse the hover effect that shows when hovering over the link/delete/tag buttons
- [ ] When viewing a post or a comment, can we see when it was made?
- [ ] Total # of saved comments, saved posts, or both may be a nice feature to have

**Comment by Logan1x** [comment]
> Have sorting on homepage/comments (start with chronological).

Are you talking about most recent to first?

**Comment by cedricium** [comment]
>most recent to first

@Logan1x yes that's correct. Due to the SQL query we use to append tags to saved things on the backend, the ordering of things is seemingly broken/random. We would want saved things to always be in reverse chronological order (most recent to oldest).
